code/extensions/ScheduledExecutionExtension.php

```diff
@@ -18,16 +20,21 @@ class ScheduledExecutionExtension extends DataExtension {
 		'ExecuteFree'			=> 'Varchar',
```

Comment:
Thanks for the work! :)

This kind of alignment isn't part of the current code style guidelines; and when I last tried to align values in this way, @tractorcow told me off.

In his book, Clean Code, Robert Martin talks a bunch about the relationship between alignment and readability. There's lots of good stuff in there, but he concludes that most of the time it doesn't help. There are exceptions, but I don't think aligning multiple array properties (like this) is one of them. If each array's keys and values were aligned then perhaps it could help...